px-data-grid-number-renderer.html

```diff
@@ -4,6 +4,30 @@
 <link rel="import" href="px-data-grid-theme.html">
 <link rel="import" href="css/px-data-grid-number-renderer-styles.html">
 
+<!--
+
+### Styling
+The following custom properties are available for styling of the number-renderer content.
+
+Custom property | Description
+'--px-data-grid-cell-number-column-overflow' | controls the overflow behavior, visible, hidden
+'--px-data-grid-cell-number-column-white-space' | controls how white-spaces are handled, normal, nowrap
+'--px-data-grid-cell-number-column-word-wrap' | controls how word-wrapping is handled normal, break-word
+'--px-data-grid-cell-number-column-text-overflow' | controls how text-overflow is handled, clip, ellipsis
+'--px-data-grid-cell-number-column-editor-align-items | control editor alignment when in edit mode, center, flex-end, flex-start'
+
+Examples:
+Make number-renderer columns wrap the content:
```

Comment:
IMO, `<style>` wrapper is also needed here, see below.

Reply:
This is just the documentation of the css-variables, the syntax is copied from here https://github.com/PredixDev/px-dropdown/blob/master/px-dropdown.html under the style-section.